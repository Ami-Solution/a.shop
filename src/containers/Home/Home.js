import React from 'react';
import { withRouter } from 'react-router-dom';

/*
  Components
 */
import Panels from '../../components/Panels';

/*
  Containers
 */
import LeftPanel from '../LeftPanel';
import Map from '../Map';

/**
 * Home container
 */
const Home = () => (
  <Panels>
    <Panels.Left>
      <LeftPanel />
    </Panels.Left>
    <Panels.Right>
      <Map />
    </Panels.Right>
  </Panels>
);

export default withRouter(Home);
