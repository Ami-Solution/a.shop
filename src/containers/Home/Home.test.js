/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import fakeStore from '../../constants/test/fakeStore';
import Home from './index';

configure({ adapter: new Adapter() });

const store = fakeStore({});

describe('Container Home', () => {
  it('should be render without crash', () => {
    const component = renderer.create(<Home store={store} />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
