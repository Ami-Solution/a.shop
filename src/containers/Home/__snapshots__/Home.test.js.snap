// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home::mount 1`] = `undefined`;

exports[`Home::shallow 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home
    acceptTerms={[Function]}
    hasEnoughDth={true}
    hasEnoughEth={true}
    hasGoodNetwork={true}
    isMetamaskInstalled={true}
    minDth="100"
    minEth={40}
    toggleTermsModal={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <SvgHome />,
        <styled.div
          all={null}
          bottom="m"
          horizontal={null}
          left={null}
          right={null}
          top={null}
          vertical={null}
>
          <styled.h1>
                    CRYPTOCURRENCY FOR SHOPS
          </styled.h1>
</styled.div>,
        <styled.div
          all={null}
          bottom={null}
          horizontal={null}
          left={null}
          right={null}
          top={null}
          vertical="l"
>
          <span>
                    Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.
          </span>
</styled.div>,
        <Message
          alignCenter={true}
          alignLeft={false}
          theme="error"
          withIcon={true}
>
          Your browser is not supported, Metamask is available on Google Chrome and Firefox only.
</Message>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "all": null,
          "bottom": "m",
          "children": <styled.h1>
            CRYPTOCURRENCY FOR SHOPS
</styled.h1>,
          "horizontal": null,
          "left": null,
          "right": null,
          "top": null,
          "vertical": null,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "CRYPTOCURRENCY FOR SHOPS",
          },
          "ref": null,
          "rendered": "CRYPTOCURRENCY FOR SHOPS",
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "all": null,
          "bottom": null,
          "children": <span>
            Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.
</span>,
          "horizontal": null,
          "left": null,
          "right": null,
          "top": null,
          "vertical": "l",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.",
          },
          "ref": null,
          "rendered": "Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.",
          "type": "span",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "alignCenter": true,
          "alignLeft": false,
          "children": "Your browser is not supported, Metamask is available on Google Chrome and Firefox only.",
          "theme": "error",
          "withIcon": true,
        },
        "ref": null,
        "rendered": "Your browser is not supported, Metamask is available on Google Chrome and Firefox only.",
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <SvgHome />,
          <styled.div
            all={null}
            bottom="m"
            horizontal={null}
            left={null}
            right={null}
            top={null}
            vertical={null}
>
            <styled.h1>
                        CRYPTOCURRENCY FOR SHOPS
            </styled.h1>
</styled.div>,
          <styled.div
            all={null}
            bottom={null}
            horizontal={null}
            left={null}
            right={null}
            top={null}
            vertical="l"
>
            <span>
                        Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.
            </span>
</styled.div>,
          <Message
            alignCenter={true}
            alignLeft={false}
            theme="error"
            withIcon={true}
>
            Your browser is not supported, Metamask is available on Google Chrome and Firefox only.
</Message>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "all": null,
            "bottom": "m",
            "children": <styled.h1>
              CRYPTOCURRENCY FOR SHOPS
</styled.h1>,
            "horizontal": null,
            "left": null,
            "right": null,
            "top": null,
            "vertical": null,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "CRYPTOCURRENCY FOR SHOPS",
            },
            "ref": null,
            "rendered": "CRYPTOCURRENCY FOR SHOPS",
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "all": null,
            "bottom": null,
            "children": <span>
              Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.
</span>,
            "horizontal": null,
            "left": null,
            "right": null,
            "top": null,
            "vertical": "l",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.",
            },
            "ref": null,
            "rendered": "Add your retail business on the Dether map. Expand your visibility in the Dether ecosystem of crypto buyers and sellers.",
            "type": "span",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "alignCenter": true,
            "alignLeft": false,
            "children": "Your browser is not supported, Metamask is available on Google Chrome and Firefox only.",
            "theme": "error",
            "withIcon": true,
          },
          "ref": null,
          "rendered": "Your browser is not supported, Metamask is available on Google Chrome and Firefox only.",
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
